README.md

```diff
@@ -112,14 +112,18 @@ Prerequisites:
 
 1. Configure your host machine system and install Docker Desktop using these instructions: https://docs.docker.com/get-docker/
 
-To run a docker container you will need to create an .env file in the root of the project containing the below variables (note the PORT, DATABASE_CONNECTION_STRING and DATABASE_NAME values should be replaced with your own values for port to use and MongoDB credentials):
+To run a docker container you will need to create an .env file in the root of the project containing the below variables (note the SECRET_KEY, PORT, DATABASE_CONNECTION_STRING, DATABASE_NAME, CLIENT_SECRET and CLIENT_ID values should be replaced with your own values for these credentials):
```

Comment:
👍 
Remember than when you update Heroku (and next exercise, when you host the app on Azure), you'll need to register another app with GitHub, and use a different client ID and secret for your remote server. Because the client id is tied to the callback URL, e.g. you need a new one with a callback URL pointing to todo-app-devops-academy.herokuapp.com instead of localhost:5000